task/fossa-scanner/0.1/fossa-scanner.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: fossa-scanner
+  namespace: tekton-pipelines
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Security
+    tekton.dev/tags: CLI, fossa
+    tekton.dev/displayName: "fossa scanner"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    fossa-cli is a zero-configuration polyglot dependency analysis tool. You can point fossa CLI at any codebase or build,
+    and it will automatically detect dependencies being used by your project.
+
+    This task can be used to scan for dependency analysis, license scanning, vulnerability scanning,
+    attribution report generation, and more on the source code in standalone mode.
+  resources:
+    inputs:
+      - name: repo
+        type: git
```

Comment:
@nistal97 PipelineResources are deprecated and will be removed soon, any reason for adding this?

Reply:
changed to v1beta taskrun resource

Reply:
@nistal97 PipelineResources as whole are deprecated. See [TEP](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md). As a replacement, we'll be using Tasks within the Pipeline.